test/clearingHouse/ClearingHouse.newMarket.ts

```diff
@@ -190,7 +186,7 @@ describe("ClearingHouse new market listing", () => {
             expect(davisPendingFundingAfter.abs()).to.be.gt(davisPendingFundingBefore.abs())
         })
 
-        it("Stop to cumulate funding after change to emergency oracle", async () => {
+        it.skip("Stop to cumulate funding after change to emergency oracle", async () => {
```

Comment:
Why we skip this test case here? Based on the new mechanism, we still might do update the priceFeed of baseToken with EmergencyPriceFeed?

Reply:
@twxia any thoughts?

Reply:
This test is to ensure that after the emergencyPriceFeed get plugged in, the funding rate should not get cumulated. 

Original oracle:
Because mark price (market twap) = index price (emergencyPriceFeed's market twap)

New oracle:
After oracle get switched to uniswap market price. 
mark price (mid(30min-twap, 15min-twap, market price)) != index price (Uniswapv3 30 minutes market twap)

Reply:
Understood that this test case is for the case when emergencyPriceFeed get plugged in. Then should we remove this case cuz we don't use emergencyPriceFeed anymore?